dev/aarch64_opt/src/arith_native_aarch64.h

```diff
@@ -31,23 +31,57 @@ extern const int16_t mlk_aarch64_zetas_mulcache_twisted_native[];
 extern const uint8_t mlk_rej_uniform_table[];
 
 #define mlk_ntt_asm MLK_NAMESPACE(ntt_asm)
-void mlk_ntt_asm(int16_t *, const int16_t *, const int16_t *);
+void mlk_ntt_asm(int16_t *p, const int16_t *, const int16_t *)
+/* This must be kept in sync with the HOL-Light specification
+ * in proofs/hol_light/arm/proofs/mlkem_ntt.ml */
+__contract__(
+  requires(memory_no_alias(p, sizeof(int16_t) * MLKEM_N))
+  requires(array_abs_bound(p, 0, MLKEM_N, 8192))
```

Comment:
The twiddle arguments should be fixed here I think, as in rej_uniform.

Reply:
Good catch. fixed - see https://github.com/pq-code-package/mlkem-native/compare/8bffbd1a4930e7c9ec10c3d25b866880187c9f48..26d81ad613cedaff66161f21acd6f913f6f67f16